web/templates/user_profile_modal.hbs

```diff
@@ -58,31 +58,50 @@
+                                        </div>
+                                        {{#if bot_owner}}
+                                        <div class="field-section bot_owner_user_field" data-field-id="{{bot_owner.user_id}}">
+                                            <div class="name">{{t "Owner" }}</div>
+                                            <div class="pill-container not-editable">
+                                                <div class="input" contenteditable="false" style="display: none;"></div>
+                                            </div>
+                                        </div>
+                                        {{/if}}
+                                    {{else}}
+                                        {{> user_custom_profile_fields profile_fields=profile_data}}
+                                    {{/if}}
+                                </div>
                             </div>
                             <div class="col-wrap col-right">
                                 <div id="avatar" {{#if user_is_guest}} class="guest-avatar" {{/if}}
                                   style="background-image: url('{{user_avatar}}');">
                                 </div>
-                            </div>
-                        </div>
-                        <div class="bottom">
-                            <div id="content">
-                                {{#if is_bot}}
-                                    <div class="field-section">
-                                        <div class="name">{{t "Bot type" }}</div>
-                                        <div class="bot_info_value">{{bot_type}}</div>
-                                    </div>
-                                    {{#if bot_owner}}
-                                    <div class="field-section bot_owner_user_field" data-field-id="{{bot_owner.user_id}}">
-                                        <div class="name">{{t "Owner" }}</div>
-                                        <div class="pill-container not-editable">
-                                            <div class="input" contenteditable="false" style="display: none;"></div>
-                                        </div>
-                                    </div>
-                                    {{/if}}
-                                {{else}}
-                                    {{> user_custom_profile_fields profile_fields=profile_data}}
-                                {{/if}}
+                                <ul class="nav nav-list-modal info_popover_actions" id="user_info_popover" data-user-id="{{user_id}}">
+                                    {{#unless spectator_view}}
+                                        {{#if can_send_private_message}}
+                                            <li>
+                                                <a tabindex="0" class="compose_private_message compose_private_message_modal">
```

Comment:
The `_modal` in class name doesn't look right, as these are just clickable elements to open compose box or message views.

Reply:
> The `_modal` in class name doesn't look right, as these are just clickable elements to open compose box or message views.

Removed the `_modal` type class name.